### PR TITLE
Clarify that champion duties extend through incubation.

### LINKED ISSUE #400: Point of contact for onboarding
While there is a clear point of contact during onboarding (the project champion and a single mailing list address), it's not super clear who to contact during incubation to help answer project-specific questions or how to do so.

I've cc'ed @brianwarner on a number of issues here and on various AMP repositories, but that feels brittle; I'm not sure Brian's always the right person, and if he received those GH notifications, and thus whether or not someone is actually working on those.

I also have specific issues that I need the CPC to answer (e.g. https://github.com/ampproject/wg-foundation-onboarding/issues/2), and I'm not sure how to go about that either.

It would be helpful to have a better idea of how to proceed here. Maybe a dedicated incubation GH team we could assign issues to? Maybe an incubation champion?

Maybe a lot of this is already organized and I'm just not aware of it.

### PR DESCRIPTION
**Pull request by mikesamuel**
Fixes #400 per @joesepi's summary of consensus

> Champion remains point of contact through incubation. Documentation
> needs to be updated to make that clear.

### CODE REVIEW

File: PROJECT_PROGRESSION.md [hunk 1]
```diff
@@ -96,14 +97,14 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
...
-- [ ] Project Charter is published on website or github
+- [ ] Publish Project Charter on website or github
 - [ ] Update legal copyright notice on project website and github
 - [ ] Add OpenJS Foundation logo to project website
 - [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
 - [ ] Transfer logomark to the OpenJS Foundation
 - [ ] If project is using crowdfunding platforms, add disclaimer to platforms
 - [ ] Identify individuals from the project to join the CPC
-- [ ] Document project and foundation contacts for:
+- [ ] The champion remains the main point of contact with the CPC, but also document project and foundation contacts for:
```
Comment (tobie) [inline PROJECT_PROGRESSION.md]:
```suggestion
- [ ] Document project and foundation contacts for:
```

Comment (mikesamuel) [reply PROJECT_PROGRESSION.md]: So rollback the change to the onboarding checklist?

File: PROJECT_PROGRESSION.md [hunk 2]
```diff
@@ -96,14 +97,14 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
 - [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection and implement appropriate tool
 - [ ] Add or Update Governance.md document (required for Impact stage)
 - [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md)
-- [ ] Project Charter is published on website or github
+- [ ] Publish Project Charter on website or github
```
Comment (molant) [inline PROJECT_PROGRESSION.md]:
```suggestion
- [ ] Publish Project Charter on website or GitHub
```

File: PROJECT_PROGRESSION.md [hunk 3]
```diff
@@ -96,14 +97,14 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
...
 - [ ] Update legal copyright notice on project website and github
```
Comment (molant) [inline PROJECT_PROGRESSION.md]:
```suggestion
- [ ] Update legal copyright notice on project website and GitHub
```

Comment (mikesamuel) [reply PROJECT_PROGRESSION.md]: Do the dashes lineup with this change?

File: PROJECT_PROGRESSION.md (hunk 1 above)
Comment (tobie) [reply PROJECT_PROGRESSION.md]: (Yes, sorry for confirming this with so much delay.)

**Pull request merged**